src/test/java/org/broadinstitute/hellbender/engine/FeatureManagerUnitTest.java

```diff
@@ -188,6 +189,33 @@ public void testHandleRequestForValidFeatureInputs() {
     }
 
     @Test
+    public void testSequenceDictionary() {
+        final ValidFeatureArgumentSource toolInstance = new ValidFeatureArgumentSource();
+
+        // Initialize two of the FeatureInput fields as they would be initialized by the argument-parsing
+        // system to simulate a run of the tool with two FeatureInputs.
+        toolInstance.variantContextFeatureInput = new FeatureInput<>(FEATURE_MANAGER_TEST_DIRECTORY + "feature_data_source_test_withSequenceDict.vcf");
+
+        final FeatureManager manager = new FeatureManager(toolInstance);
+        final List<SAMSequenceDictionary> dicts = manager.getSequenceDictionaries();
+        Assert.assertEquals(dicts.size(), 1);
+        Assert.assertEquals(dicts.get(0).getSequences().size(), 84);
+    }
+
+    @Test
+    public void testEmptySequenceDictionary() {
+        final ValidFeatureArgumentSource toolInstance = new ValidFeatureArgumentSource();
+
+        // Initialize two of the FeatureInput fields as they would be initialized by the argument-parsing
```

Comment:
Here too -- you are only initializing one of the FeatureInput fields here, aren't you?